crypto/crypto.go

```diff
@@ -59,9 +62,14 @@ func rawTransactionBytesToSign(tx types.Transaction) ([]byte) {
 	return bytes.Join(msgParts, nil)
 }
 
+// rawHash computes Sum512_256 hash from raw bytes
+func rawHash(input []byte) types.Digest {
```

Comment:
Since we really use only one hash function, I believe that there is no need to wrap it.

Reply:
Done